src/sugar3/graphics/progressicon.py

```diff
@@ -14,86 +14,154 @@
 # License along with this library; if not, write to the
 # Free Software Foundation, Inc., 59 Temple Place - Suite 330,
 # Boston, MA 02111-1307, USA.
+'''
+The progressicon module provides a progress icon; a widget like progress
+bar which shows progress of a task 
+'''
 
 from gi.repository import Gtk
 from sugar3.graphics.icon import get_surface
 from sugar3.graphics import style
-
-
-class ProgressIcon(Gtk.DrawingArea):
-    """Display the progress filling the icon.
-
-    This class is compatible with the sugar3.graphics.icon.Icon class.
-
-    Call update(progress) with the new progress to update the icon.
-
-    The direction defaults to 'vertical', in which case the icon is
-    filled from bottom to top.  If direction is set to 'horizontal',
-    it will be filled from right to left or from left to right,
-    depending on the system's language RTL setting.
-
-    """
-    def __init__(self, icon_name, pixel_size, stroke_color, fill_color,
+ 
+class ProgressIcon(Gtk.DrawingArea): '''UI interface for Progress Icon.'''
+ 
+    '''
+      Display the progress filling the icon.
+      This class is compatible with the sugar3.graphics.icon.Icon class.
+      The direction defaults to 'vertical', in which case the icon is
+      filled from bottom to top.  If direction is set to 'horizontal',
+      it will be filled from right to left or from left to right,
+      depending on the system's language RTL setting.
+      
+    Parameters:
+    
+      pixel_size - sets the icon size [e.g. pixel_size=style.LARGE_ICON_SIZE]
```

Comment:
Please use the same syntax as used in the example on the wiki tutorial page (linked in gci task)